src/utils/actions.js

```diff
@@ -1,29 +1,29 @@
 import fetch from "cross-fetch";
 
-import {message} from "antd";
+import { message } from "antd";
 
-import {BENTO_PUBLIC_URL, BENTO_URL, IDP_BASE_URL} from "../config";
+import { BENTO_PUBLIC_URL, BENTO_URL, IDP_BASE_URL } from "../config";
 
-export const basicAction = t => () => ({type: t});
+export const basicAction = (t) => () => ({ type: t });
```

Comment:
why are you applying prettier to bento web? it makes the diffs very hard to read.

Reply:
i'll just do a whole-pass prettier run after this; i'd rather have readable PR diffs than blame.